src/lib/ngsi/AttributeList.cpp

```diff
@@ -110,6 +112,25 @@ void AttributeList::release(void)
 
 /* ****************************************************************************
 *
+* lookup - 
+*/
+bool AttributeList::lookup(const std::string& attributeName)
```

Comment:
Nice method... probably some existing parts of the code (I'm thinking in mongoBackend) are doing this and we can reuse.

NTC (just a thought)